app/Bus/Commands/Incident/ReportIncidentCommand.php

```diff
@@ -63,6 +63,13 @@
     public $notify;
 
     /**
+     * Whether to stick the incident on top.
+     *
+     * @var bool
+     */
+    public $sticked;
```

Comment:
Dude, "stickied" is a real word http://www.collinsdictionary.com/dictionary/english/stickied